### PR TITLE
Create coinflip.cc.lua

### PR DESCRIPTION
**Pull request by DaviiD1337**

### CODE REVIEW

File: fun/coinflip.cc.lua [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{{/*
+	This command is a game of heads or tails where users can play.
+
+	Usage: coinflip <heads/tails> <bet>
+
+	Recommended trigger: "coinflip"
+	Trigger type: Command
+
+	Credits:
+	Joe L. <https://github.com/Jo3-L>
```
Comment (Jo3-L) [inline fun/coinflip.cc.lua:10]: :cereal:

**Review by Jo3-L — CHANGES_REQUESTED** [review]

**Comment by Jo3-L** [comment]
Please use the `.go.tmpl` file extension as of latest commit. Thanks!

File: fun/coinflip.cc.lua (hunk 1 above)
Comment (DaviiD1337) [reply fun/coinflip.cc.lua]: :cereal:

**Comment by Jo3-L** [comment]
👌

**Pull request merged**